esignet-dev.properties

```diff
@@ -116,8 +116,6 @@ mosip.esignet.authenticator.ida-env=Developer
 mosip.esignet.authenticator.ida.otp-channels=email,phone
 
 # Mock IDA integration props
-mosip.esignet.mock.authenticator.persona-repo=/home/mosip/mockida
-mosip.esignet.mock.authenticator.policy-repo=/home/mosip/mockida
 mosip.esignet.mock.authenticator.claims-mapping-file=/home/mosip/mockida/claims_attributes_mapping.json
```

Comment:
i think this property is also not required. can you pls cross check.

Reply:
Commented all three properties.